extension/llm/custom_ops/op_sdpa.cpp

```diff
@@ -382,7 +382,7 @@ void cpu_flash_attention(
       /* qk_sum */ qSplitSize +
       /* dst    */ qSplitSize * headSize;
 
-  int64_t size_bytes = size_per_thread * num_thread * query.element_size();
+  int64_t size_bytes = size_per_thread * num_thread * query.element_size() * 4;
```

Comment:
Is *4 for fp32?

Reply:
oh really good catch. Thats was left over from local debug. need to remove

Reply:
please finalize so that @spalatinate has the correct version

Reply:
here is the fix pr https://github.com/pytorch/executorch/pull/9492